RELEASE_PROCEDURE.md

```diff
@@ -11,55 +11,30 @@
        - Version bumping: In stellargraph/version.py, change version from “X.X.Xb” to “X.X.X”. E.g. version=”0.2.0b” to version=”0.2.0”
        - Update Changelog section header to point to specific version tag instead of `HEAD`. Note: this link will be broken until the tag is pushed later.
        - Remove "Full Changelog" link from Changelog.
+       - Update version in `meta.yaml`
```

Comment:
Maybe this could be included as "Version bumping: In `stellargraph/version.py` and `meta.yaml`, ..." or similar.